versions/3.0.3.md

```diff
@@ -427,14 +429,16 @@ servers:
 
 #### <a name="serverVariableObject"></a>Server Variable Object
 
-An object representing a Server Variable for server URL template substitution.
+An object representing a Server Variable for URL template substitution in a containing [Server Object](#serverObject).
+
+To form an addressable server URL, the API consumer MUST substitute a valid value for each variable expression in the Server Object's `url` template. The API consumer MAY assign Server Variable values from user input, configuration settings, or any other source. If the API consumer does not have an assigned value for a given server variable, it MUST substitute the provided `default` value.
```

Comment:
Up-front summary of the basic workings.

Reply:
The second paragraph is great content for a best practices documentation.  I don't believe the spec the right place to include detailed usage guidance.  We need to limit the text to constraints that tooling must/should respect.